### PR TITLE
Add api docs and explorer

### PR DESCRIPTION
**Pull request by tspenov**
#### Summary
* Allow `graphiql` API explorer on stage & prod
* Add a simple page with API examples

the url on PROD  will be  : `https://sanbase-low.santiment.net/apidocs`

[//]: # (#### Screenshots)
image
image

### CODE REVIEW

File: lib/sanbase_web/router.ex [hunk 1]
```diff
@@ -38,21 +38,21 @@ defmodule SanbaseWeb.Router do
...
-        max_complexity: 5000
-      )
-    end
+    forward(
+      "/graphiql",
+      Absinthe.Plug.GraphiQL,
+      schema: SanbaseWeb.Graphql.Schema,
+      analyze_complexity: true,
+      max_complexity: 5000,
+      interface: :simple
```
Comment (tspenov) [inline lib/sanbase_web/router.ex]: Only the `simple` interface handles correctly the url params which we use in `Run in explorer` and It misses the option to add headers. The advanced interface I think has some problems with the `localStorage` and does not show correctly the url params

File: lib/sanbase_web/router.ex [hunk 2]
```diff
@@ -38,21 +38,21 @@ defmodule SanbaseWeb.Router do
...
+    )
   end
 
   scope "/", SanbaseWeb do
     pipe_through(:browser)
 
     get("/consent", RootController, :consent)
+    get("/apidocs", ApiDocsController, :api_examples)
```
Comment (IvanIvanoff) [inline lib/sanbase_web/router.ex]: shouldn't this be `apiexamples`?  Also in the previous PR there was `/apiexplorer` if I'm not mistaken?

**Comment by tspenov** [comment]
@IvanIvanoff 
the url when deployed to production will be:
`https://sanbase-low.santiment.net/apidocs` - for the docs and examples
`https://sanbase-low.santiment.net/apiexplorer` - for the graphiql explorer

File: lib/sanbase_web/router.ex (hunk 2 above)
Comment (tspenov) [reply lib/sanbase_web/router.ex]: updated things a bit - should be good now

**Comment by IvanIvanoff** [comment]
One more thing - We should add links to the apidocs somewhere (in the footer most likely)

**Pull request merged**